CeeFM/Decideify
# Delete Suggestion Button and Confirmation Modal

**Description**
Delete button is now functional for saved suggestions in a user's database

**Type of Change**


**Testing Instructions**

- Clone this project’s repository to your machine
  -If you’ve already done this and are currently working on a different branch of the project, be sure to `git add` and `git commit` whatever you’re working on on your branch, before you attempt to switch branches
  -Then `git checkout main` to switch to the project’s main branch)
- Navigate to the root directory for this repo (yourParentFolderName/Decideify)
- `git fetch -a` to retrieve the current remote-tracked branches for the project, you should see a branch named `cm-suggestions` as one of the available branches
- `git checkout cm-suggestions` to switch to my branch
- If you don't see the most recent version of my files, `git pull origin cm-suggestions`
- `Start Decideify.sln` to open the project in Visual Studio
- If you haven’t created the Decideify database, please do so in Visual Studio and use the following SQL scripts to populate it accordingly:
  - Start a new query in your database and copy in this SQL script before running to fill out tables and columns: [First SQL Script for Creating DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Create_DB.sql)
  - ctrl + a and delete what is in that query and replace it with this SQL script before running to seed the database: [Second SQL Script for Seeding DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Seed_Users_DB.sql)

- Click the execute with debugger solid green button to launch the app
- Log in as coolin@gmail.com with password thistbh
- On every content hub page and on the mylibrary.js component / page, you should see a red X button in the upper left corner of each suggestion.
- Clicking it should open a modal that asks you to confirm you want to delete that suggestion.
- Clicking cancel will close the modal and no changes will be made to the database
- Clicking confirm delete will delete that suggestion from the user's saved suggestions database, and will reload the page to visually remove said suggestion from the carousel of displayed suggestions.

Checklist
